### PR TITLE
Now shows temporary todo list

### PR DESCRIPTION
**Pull request by blaytenshi**
#3 Show Temporary Todo list
- Added TodoList component
- Added TodoItem component
- Tidied up README.MD

### CODE REVIEW

File: src/Components/TodoItem.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <slot name="description"></slot>
+    </div>
+  </li>
+</template>
+
+<script>
+export default {
+  name: "TodoItem",
+  props: {
+    item: {
```
Comment (BenFranzi) [inline src/Components/TodoItem.vue:16]: It seems like the props are unused as we are using slots <insert meaningful peer review comments here>

Comment (BenFranzi) [reply src/Components/TodoItem.vue:16]: Is this what you do in peer reviews?

**Review by BenFranzi — APPROVED** [review]
passing props to the todo item seems not needed because we're using slots to pass the values through. But eh

File: src/Components/TodoItem.vue (hunk 1 above)
Comment (blaytenshi) [reply src/Components/TodoItem.vue:16]: Yeah sure you can fix it in a new ticket :p

**Pull request merged**